setup.cfg

```diff
@@ -132,3 +132,7 @@ import_heading_stdlib = stdlib
 import_heading_thirdparty = third party
 ignore_comments = False
 honor_noqa = True
+
+[darglint]
+style=google # Default style is Google but we opt to be explicit about it
```

Comment:
Q: Those are all the options that SyftCore added for ```darglint```?

Reply:
Yes